.gitignore

```diff
@@ -4,6 +4,6 @@
 
 /node_modules/
 
-/package-json.lock
+/package-lock.json
```

Comment:
@MarshallOfSound This change is introduced in https://github.com/electron/electron-osx-sign/pull/202/files#diff-a084b794bc0759e7a6b77810e01874f2R7